src/sections/2022/section-3.mdx

```diff
@@ -14,7 +15,22 @@ import Quote from "../../components/2022/quote"
     <p>We asked subscribers about their experiences using design systems. We especially wanted to know if their needs were met and what challenges they faced.</p>
   </SectionHeader>
   
-  <div>PIE CHART</div>
+  <Grid className="util-margin-top-1xl">
+    <GridCell spanMD={7} spanLG={9}>
+      <h3>The design system meets my needs.</h3>
+    </GridCell>
+    <GridCell spanMD={3}>
+      <div class="cmp-pie-chart-caption">
+        <div class="cmp-pie-chart-caption__content">
+          <p class="cmp-type-body-default">Of design system subscriber respondents, 65% said their design system met their&nbsp;needs.</p>
```

Comment:
I'll bet all 61% of people who are able to contribute to their system fit inside this 65% number. Even if the system isn't perfect for everyone, the ability to contribute creates a positive working relationship with it.